suite/sanity/autoscale.go

```diff
@@ -15,19 +15,20 @@ func autoscale(p interface{}) (gravity.TestFunc, error) {
 			g.Maybe("destroy", cluster.Destroy())
 		}()
 
-		g.OK("status", g.Status(cluster.Nodes))
+		g.OK("wait for active status on masters", g.WaitForActiveStatus(cluster.Nodes))
 		g.OK("time sync", g.CheckTimeSync(cluster.Nodes))
 
 		// Scale Up
 		workers, err := g.AutoScale(3)
 		g.OK("asg-up", err)
-		g.OK("status-masters", g.Status(cluster.Nodes))
-		g.OK("status-workers", g.Status(workers))
+		g.OK("wait for active status on masters", g.WaitForActiveStatus(cluster.Nodes))
+		g.OK("wait for active status on asg workers", g.WaitForActiveStatus(workers))
 
 		// Scale Down
 		workers, err = g.AutoScale(1)
 		g.OK("asg-down", err)
-		g.OK("status-masters", g.Status(cluster.Nodes))
-		g.OK("status-workers", g.Status(workers))
+		g.OK("wait for active status on masters", g.WaitForActiveStatus(cluster.Nodes))
+		_, err = g.Status(workers)
```

Comment:
This no longer waits for `state != degraded`.  We don't run this test presently, but if we did this could be a breaking change.

Reply:
I'm going to ignore this until we have occasion to bring autoscale back.